### PR TITLE
Move membersList to it's own component

### LINKED ISSUE #17: Update members.js to use the MembersItem component
update pages/members.js to use the MemberItem component

### PR DESCRIPTION
**Pull request by galiotoweb**
## Description

Move the formatting for the list of members to it's own component instead of it all being in the members page file.
No styling was changed.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation content changes
- [ ] Other (please describe)

## List any open issues this change will close
This closes #17

## Describe how to test/validate these changes
Build repo locally

## Screenshots (if appropriate)
Screenshot 2020-10-24 members page

### CODE REVIEW

**Comment by galiotoweb** [comment]
Happy Halloween and Happy Hacktoberfest! 🎃 👩🏻‍💻

Tagging @alexxmitchell for a review

File: src/components/membersItem.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+const MembersItem = ({ name, image, location, techs, hobbies, links }) => (
+  <div>
+    {image && <img src={image} alt={name + `'s beautiful face`} style={{ maxWidth: 200 }}
+      onError={(event) => { event.target.onerror = null; event.target.src = "https://avatars3.githubusercontent.com/u/13358086?s=200&v=4" }}
```
Comment (alexxmitchell) [inline src/components/membersItem.js]: @galiotoweb Can you pull this out to a function similar to how it was before? It will help with readability

**Review by alexxmitchell — CHANGES_REQUESTED** [review]
As well please run `prettier --write "src/pages/members.js" "src/pages/membersItem.js"` in your terminal to format these files

**Comment by galiotoweb** [comment]
Moved the onError to its own line and formatted the files.

Prettier stopped working in my VS Code, and I didn't realize it, so thanks for the call out.

**Review by alexxmitchell — APPROVED** [review]
LGTM 👍

**Pull request merged**